package.json

```diff
@@ -1,7 +1,18 @@
 {
-  "name": "design-system-prototypes",
-  "private": true,
   "version": "0.0.0",
+  "author": "City of Amsterdam",
+  "description": "Prototyping environment for the Amsterdam Design System",
+  "license": "EUPL-1.2",
```

Comment:
If we specify this license, we probably also need to add the license file to the repo

Reply:
Yea, removed this line again.